docs/design/test-results.md

```diff
@@ -0,0 +1,143 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Externally accessible Kubernetes test results
+
+## Objective
+
+Make kubernetes test results accessible to anyone on the internet
+
+## Background
+
+For the time being Google runs most of the testing for kubernetes. As a result,
+developers working inside Google have access to better test result information
+than everyone else. We want to change that. We want the best tools for looking
+at test results inside and outside Google to be one and the same.
+
+Specifically we have two primary ways of looking at test results:
+
+1. A summary from the past twenty-four hours
+2. A view of recent test results updated every thirty minutes
+
+Option 1 is now available to everyone at
+http://storage.googleapis.com/kubernetes-test-history/static/index.html
+
+Option 2 is available to people inside of Google, but we will make it available
+to everyone in time for the release of 1.3
+
+![test result view](../images/test-results.png)
+
+## Detailed design
+
+We will take the tool we have access to internally and put it on a public
+website. Then we will update the tool to collect information from multiple
+sources.
+
+### Storing test result information
+
+* We will associate each jenkins jobs with a tab name
+* We will associate each jenkins job with a GCS prefix
+  * Example: gs://kubernetes-jenkins/logs/kubernetes-e2e-gce/
```

Comment:
each of these has a `latest-build.txt` file indicating the most recent build number.